content/articles/spring-cloud-k8/index.md

```diff
@@ -0,0 +1,234 @@
+### Tutorial overview
+
+In this tutorial, I'll show you how to use Kubernetes and Spring cloud to build your microservice application. It'll be an interesting project that will walk you through the whole process of integrating Kubernetes with Spring Boot applications.
```

Comment:
```suggestion
### Tutorial overview
In this tutorial, we will learn how to use Kubernetes and Spring cloud to build a micro-service application and how to integrate it with Spring Boot.
```
Please remove the blank line(s) between a heading and the paragraph that follows it. Do this all through the article.

Reply:
@Ericgacoki ,fixed.